assets/src/edit-story/app/media/utils/useUploadVideoFrame.js

```diff
@@ -54,11 +56,13 @@ function useUploadVideoFrame({ updateMediaElement }) {
         featured_media: posterId,
```

Comment:
The response from this api call is the data you need for the resource. Can we reuse it?

Reply:
This would get you the resource in the correct format.
```
  const video = await updateMedia(videoId, {
        featured_media: posterId,
        post: storyId,
  });
  const resource = getResourceFromAttachment(video);
```

Reply:
![Screenshot from 2020-04-01 12-20-57](https://user-images.githubusercontent.com/237508/78131612-4344bb80-7413-11ea-803f-7667310a3f11.png)
The resource even gives you the poster etc.

Reply:
I think it's not appropriate to use this resource to update the elements resource. It seems it's not same like the width and the height.